### PR TITLE
STIJ-155: Small css fixes for form elements.

### PR DESCRIPTION
**Pull request by meirege**

### CODE REVIEW

File: components/21-atoms/label/_label.scss [hunk 1]
```diff
@@ -4,6 +4,7 @@ label {
 
     .label-optional {
       color: $label-optional-color;
+
```
Comment (delrueba) [inline components/21-atoms/label/_label.scss]: line

File: components/21-atoms/inputs/input-multiselect-chosen/_input-multiselect-chosen.scss [hunk 2]
```diff
@@ -1,4 +1,5 @@
 .chosen-container {
+  width: 100% !important; // sass-lint:disable-line no-important
```
Comment (delrueba) [inline components/21-atoms/inputs/input-multiselect-chosen/_input-multiselect-chosen.scss]: There are now six !importants in this file.

Comment (meirege) [reply components/21-atoms/inputs/input-multiselect-chosen/_input-multiselect-chosen.scss]: That's because we are overriding JS generated code.

Comment (delrueba) [reply components/21-atoms/inputs/input-multiselect-chosen/_input-multiselect-chosen.scss]: Assuming we have no control over de JS in question: OK.

**Pull request merged**